handout/a11y/why-a11y.md

```diff
@@ -0,0 +1,22 @@
+# Why Accessibility
+
+While making websites accessible can lead to more time spent in development, there are many reasons why you should be making your application accessible.
+
+## Reaching out to everyone
+
+Multiple studies suggest that around 15-20% of the population is suffering from a disability of some kind<sup>1</sup>. In comparison, that number is higher than any single browser demographic currently, other than Chrome<sup>2</sup>. Not considering those users when developing an application means excluding a large number of people from being able use it comfortable or at all.
```

Comment:
`Suffering` is a word disability advocates prefer to avoid - many of them don't consider themselves to be suffering. Change to `living with a disability`. You could also provide a concrete example to go with that statistic of someone with bad eyesight who just wants to be able to make the fonts bigger without everything breaking.